tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/DataManagement/DashboardTests.cs

```diff
@@ -67,5 +69,39 @@ void AssertHaveCard(string testId, bool expectCard)
                 }
             }
         }
+
+        [Fact]
+        public async Task Get_UnpublishedVenueUploads()
+        {
+            // Arrange
+            var provider = await TestData.CreateProvider(
+                apprenticeshipQAStatus: ApprenticeshipQAStatus.NotStarted,
+                providerType: ProviderType.FE);
+
+            //Create some venue upload rows to test new data in UI
+            var (venueUpload, _) = await TestData.CreateVenueUpload(providerId: provider.ProviderId, createdBy: User.ToUserInfo(), uploadStatus: UploadStatus.ProcessedWithErrors,
+                rowBuilder =>
+                {
+                    rowBuilder.AddRow(record => record.IsValid = false);
+                    rowBuilder.AddRow(record => record.IsValid = false);
+                    rowBuilder.AddRow(record => record.IsValid = false);
+                    rowBuilder.AddRow(record => record.IsValid = false);
+                });
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"/data-upload?providerId={provider.ProviderId}");
+
+            // Act
+            var response = await HttpClient.SendAsync(request);
+
+            // Assert
+            response.StatusCode.Should().Be(HttpStatusCode.OK);
+
+            var doc = await response.GetDocument();
+            using (new AssertionScope())
+            {
+                doc.GetElementByTestId("UnpublishedVenueCount").TextContent.Should().Be("4");
+            }
+
+        }
```

Comment:
Could use a test that the correct link is displayed when there is a Venue Upload in progress vs when there's not.